commands/core/config.drush.inc

```diff
@@ -146,6 +156,31 @@ function config_drush_command() {
       'drush --bg config-edit image.style.large' => 'Return to shell prompt as soon as the editor window opens.',
     ),
   );
+  $items['config-merge'] = array(
+    'description' => 'Combine configuration data from this site with the configuration data of another site; if there are conflicts, open an interactive three-way merge tool comparing the changes with the base revision.',
```

Comment:
We should add an abbreviated description in hook_help. That gets used in the command listing, as opposed to this full description which would be used for single command help.